data/nlu.yml

```diff
@@ -90,3 +90,18 @@ nlu:
     - are you a human?
     - am I talking to a bot?
     - am I talking to a human?
+
+
+- intent: hello_world
+  examples: |
+    – Say hello world
+    – print hello world
+    – utter hello world for me
+
+- intent: covid
+  examples: |
```

Comment:
this is great -- I would throw in more examples and keywords for the bot to better understand the "covid" intent.

for instance:
```
- covid
- corona
- coronavirus
- covid-19
- sars-cov-2
- coronavirus prevention
- coronavirus symptoms
- how do I know if I have covid-19
- what should I do if I have covid-19
- covid testing
- covid vaccinations
```

etc